ivy_tests/test_ivy/test_frontends/test_mindspore/test_ops/test_function/test_mindspore_nn_func.py

```diff
@@ -77,3 +77,45 @@
 #         on_device=on_device,
 #         x=x[0],
 #     )
+
+# kl_div
```

Comment:
Hey! Can you please uncomment your test and show the results of your local run? Thanks!

Reply:
@kurshakuz I commented out all the test code since there is no support for the MindSpore framework yet, however, the logic is correct.